components/CreateTrip/index.js

```diff
@@ -0,0 +1,5 @@
+import { StyledLink } from "./CreateTrip.styled";
+
+export default function CreateTrip() {
+  return <StyledLink href="/forms/TripForm">+</StyledLink>;
+}
```

Comment:
I suggest to include an `aria-label` to make this button more accessible.
```suggestion
  return <StyledLink href="/forms/TripForm" aria-label="Add a new trip to your travel list">+</StyledLink>;
}
```